flask_app.py

```diff
@@ -156,6 +156,28 @@ def reddit_callback():
     }
     return build_template_response(None, ERROR_PAGE, template_params)
 
+@app.route(DISCORD_CALLBACK_ROUTE)
+def discord_callback():
+    error = request.args.get('error', '')
+    if error:
+        return "Error: " + error
+    state = request.args.get('state', '')
+    if not is_valid_state(state):
+        # Uh-oh, this request wasn't started by us!
+        log.error(f"unknown,403 from Discord Auth API. WTF bro.")
+        abort(403)
+    code = request.args.get('code')
+    access_token = get_discord_token(code)  
+    if access_token:
+        response = make_response(redirect('/'))
+        response.set_cookie('a', access_token.encode())
+        return response
+    template_params = {
+        "error_message": f"Sorry, there was a problem authenticating you. Please contact the devs on Discord.",
+        "link": "https://discord.gg/xTqU2UmmU5"
```

Comment:
Ope this is the Michigan Risk Discord. Not sure what would be better. Maybe we get a dedicated channel on the main Risk Discord. Or better yet, we move this to a constant/config value so each deployment can set their own support channel.

Reply:
Added GOOD_GUYS_DISCORD_LINK in .env